qiskit/synthesis/unitary/qsd.py

```diff
@@ -63,21 +65,17 @@ def qs_decomposition(
 
     This decomposition is described in arXiv:quant-ph/0406176.
 
-    Arguments:
-       mat (ndarray): unitary matrix to decompose
-       opt_a1 (bool): whether to try optimization A.1 from Shende. This should eliminate 1 cnot
-          per call. If True CZ gates are left in the output. If desired these can be further decomposed
-          to CX.
-       opt_a2 (bool): whether to try optimization A.2 from Shende. This decomposes two qubit
-          unitaries into a diagonal gate and a two cx unitary and reduces overal cx count by
-          4^(n-2) - 1.
-       decomposer_1q (None or Object): optional 1Q decomposer. If None, uses
-          :class:`~qiskit.quantum_info.synthesis.one_qubit_decomposer.OneQubitEulerDecomser`
-       decomposer_2q (None or Object): optional 2Q decomposer. If None, uses
-          :class:`~qiskit.quantum_info.synthesis.two_qubit_decomposer.two_qubit_cnot_decompose
+    Args:
+        mat (ndarray): unitary matrix to decompose
+        opt_a1 (bool): whether to try optimization A.1 from Shende. This should eliminate 1 cx per call.
+            If True CZ gates are left in the output. If desired these can be further decomposed to CX.
+        opt_a2 (bool): whether to try optimization A.2 from Shende. This decomposes two qubit unitaries
+            into a diagonal gate and a two cx unitary and reduces overall cx count by 4^(n-2) - 1.
```

Comment:
If you wanted you could wrap this in math to make it render nicely in the docs
```suggestion
            into a diagonal gate and a two cx unitary and reduces overall cx count by :math:`4^(n-2) - 1`.
```